pkg/lib/cockpit.d.ts

```diff
@@ -145,6 +150,12 @@ declare module 'cockpit' {
         environ?: string[];
         pty?: boolean;
         superuser?: "try" | "require";
+
+        // for remote channels
+        host?: string;
```

Comment:
This seems wrong.  These options are applicable to all channels.  Probably this needs to be turned into a base class `ChannelOptions`? that all other specific channel types extend from...

Reply:
There is a `ChannelOptions` already, but it has `payload`, i.e. is meant to open an arbitrary channel. I'll split it up into a base class and one for a generic channel.